code/_onclick/item_attack.dm

```diff
@@ -65,8 +65,10 @@
 	/////////////////////////
 	user.lastattacked = M
 	M.lastattacker = user
-	if(force > 0)//HONK HONK HONK HONK SHUT UP, GODDAMMIT
+	if(force > 0)
 		add_logs(user, M, "attacked", name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(damtype)])")
+	else
+		add_logs(user, M, "attacked", name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(damtype)])", admin = 0)
```

Comment:
how about instead of the if statement, just use 

`add_logs(user, M, "attacked", name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(damtype)])", admin = (force > 0))`